app/main.py

```diff
@@ -2,0 +1,24 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
It's a good practice to use type hinting for the return type of a function, but it would be better to specify the type of the elements in the list of friends. For example, if friend is a string, use `List[str]`.